tools/release/java/src/main/java/com/ibm/icu/dev/tools/docs/StableAPI.java

```diff
@@ -48,72 +48,64 @@
 import org.xml.sax.InputSource;
 import org.xml.sax.SAXException;
 
-
 /**
- A utility to report the status change between two ICU releases
-
-To use the utility
-1. Generate the XML files
-    (put the two ICU releases on your machine ^_^ )
-    (generate 'Doxygen' file on Windows platform with Cygwin's help)
-    Edit the generated 'Doxygen' file under ICU4C source directory
-    a) GENERATE_XML           = YES
-    b) Sync the ALIASES definiation
-       (For example, copy the ALIASES defination from ICU 3.6
-       Doxygen file to ICU 3.4 Doxygen file.)
-    c) gerenate the XML files
-2. Build the tool
-    Download Apache Xerces Java Parser
-    Build this file with the library
-3. Edit the api-report-config.xml file & Change the file according your real configuration
-4. Run the tool to generate the report.
-
+ * A utility to report the status change between two ICU releases
```

Comment:
Please undo the reformatting of this block -- makes it unreadable.

Reply:
done